src/session.ts

```diff
@@ -102,7 +102,6 @@ export class DebugSession implements IDebugger.ISession {
    * Start a new debug session
    */
   async start(): Promise<void> {
-    await this._ready();
```

Comment:
Those shouldn't be needed, as there is already a call to `this._ready()` in `sendRequest`.